src/css/tabs/auxiliary.css

```
@@ -25,21 +25,30 @@
 
+.tab-auxiliary .mode.disabled .info {
+    background: var(--error);
+    color: white;
+}
+
+.tab-auxiliary .mode.on:nth-child(odd) .info {
+    background: var(--accent);
+}
+
 .tab-auxiliary .mode.off:nth-child(odd) .info {
     background: #cb4747;
```

Stylelint (17.14.0)

[error] Duplicate property "background" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)